cpp/inc/bond/protocol/detail/rapidjson_helper.h

```diff
@@ -63,24 +62,53 @@ class RapidJsonInputStream
 public:
     typedef char Ch;
 
-    RapidJsonInputStream(typename boost::call_traits<Buffer>::reference input)
-        : input(&input),
+    explicit RapidJsonInputStream(const Buffer& buf)
+        : input(buf),
```

Comment:
Minor: rename the variables to have `_` prefix since you are updating this file anyways?